website/docs/r/lb_listener.html.markdown

```diff
@@ -85,6 +85,67 @@ resource "aws_lb_listener" "front_end" {
 }
 ```
 
+### Authenticate-cognito Action
+
+```hcl
+resource "aws_lb" "front_end" {
+  # ...
+}
+
+resource "aws_cognito_user_pool" "pool" {
+  # ...
+}
+
+resource "aws_cognito_user_pool_client" "client" {
+  # ...
+}
+
+resource "aws_cognito_user_pool_domain" "domain" {
+  # ...
+}
+
+resource "aws_lb_listener" "front_end" {
+  load_balancer_arn = "${aws_lb.front_end.arn}"
+  port              = "80"
+  protocol          = "HTTP"
+
+  default_action {
```

Comment:
Doesn't this require two `default_action` with `order` set? I believe the second action is required for what to do with the request after its authenticated.

Reply:
Good point! 👀I haven't tested those example. 🤦‍♂️  I'll do this now.

Reply:
You are absolutely right. 🙌 I missed this in the tests (and my local setup is a bit different).

Should be 🆗 now.